common/djangoapps/student/migrations/0001_initial.py

```diff
@@ -230,7 +230,6 @@ class Migration(migrations.Migration):
                 ('bio', models.CharField(max_length=3000, null=True, blank=True)),
                 ('profile_image_uploaded_at', models.DateTimeField(null=True)),
                 ('user', models.OneToOneField(related_name='profile', to=settings.AUTH_USER_MODEL)),
-                ('nonregistered', models.BooleanField(default=False)),
```

Comment:
Restore this line, then once you've removed `nonregistered` from the model file, run `manage.py makemigrations`